recipes/regain/LICENSE

```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2015-2022, conda-forge
+BSD 3-Clause License
```

Comment:
It looks like you've *moved* the top-level `LICENSE.txt`? You shouldn't touch anything outside `recipes/regain` so you'll want to restore that file.

Reply:
It looks like the actual license is included in the sdist therefore you don't need to include a copy here so I'd actually delete the version in this recipe folder.

You only need to include a copy of the license if it's not packaged in the sdist.